tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -440,7 +262,7 @@ TEST(TRT_TensorOrWeights_Test, Basic) {
       EXPECT_EQ(false, ptr->is_tensor());
       EXPECT_EQ(true, ptr->is_weights());
       EXPECT_TRUE(TrtShapedWeightsEquals(weights, ptr->weights()));
-      ExpectTrtDimsEqualsArray({}, ptr->GetTrtDims());
+      EXPECT_THAT(ptr->GetTrtDims(), DimsAreArray({}));
```

Comment:
third_party/tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc:264:51: error: call to implicitly-deleted default constructor of 'tensorflow::tensorrt::convert::DimsAreArrayHelperMatcherP<std::vector<int>> &&'
      EXPECT_THAT(ptr->GetTrtDims(), DimsAreArray({}));

There are similar cases in other places.

Reply:
I am not sure why you can't reproduce this, but replacing the empty vector "{}" by "empty_vector" defined as follow fix the problem.
std::vector<int> empty_vector;

Reply:
Sure I can do that. It's pretty clear from all the CI builds that the public Tensorflow build does not have this problem. The only builds failing are Windows, which fails on some unrelated issue, and the "feedback/copybara", which I cannot view.

Reply:
Do you have "-fno-elide-constructors" set?